mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -64,6 +64,14 @@ data class ChessDocument(
     val moves: List<String>? = null,
     val whiteId: String? = null,
     val blackId: String? = null,
+    val lastUpdatedAt: Long? = null,
+    val metadata: ChessMetadata? = null
+)
+
+data class ChessMetadata(
+    val status: String? = null, // whiteWon, blackWon, draw
```

Comment:
It's worth specifying these magic values as constants, maybe in a `companion object` for `ChessMetadata`

Reply:
yea I think this is better. It felt a bit like random strings in the code